MELA/interface/Mela.h

```diff
@@ -96,6 +96,7 @@ class Mela{
   void getConstant(float& prob); // <ME> constants
   void getPAux(float& prob); // SuperProb
 
+  const TVar::event_scales_type& getRenFacScaleMode();
```

Comment:
Should be

`const TVar::event_scales_type& getRenFacScaleMode() **const**;`

(Similar everywhere else)